README.md

```diff
@@ -137,71 +345,33 @@ Analogously, `>>> 0` always returns a Number in uint32 range, throwing as `>>>`
-
-### Allowing mixed operands
-
-We could allow mixed operands like `1 + 1n` returning `2` (a Number). It would lose precision, be harder to implement with as high performance, and not generalize well to user-defined types, but it would follow the well-worn path of many other programming languages, which have just let users deal with these issues.
-
-### Leave out TypedArrays and DataView methods for now
-
-One possibility would be to wait until a potential future Int64/Uint64 proposal is created for TypedArray classes and DataView methods. However, they are included here, following the pattern of existing TypedArrays which return Numbers--a more general type than the contents, but one which accurately represents it.
-
-## Left for future proposals
-
-### Function and constant library ([bug](https://github.com/tc39/proposal-bigint/issues/20))
-
-It would be reasonable to add integer-related mathematical library functions, especially those which could be more efficiently based on instructions found on CPUs likely to be used by implementations. This includes:
-- Bitcast between BigInt and Number
-- Find first set/unset bit
-- Popcount
-- Find most significant set/unset bit
-- Convenience functions for doing arithmetic in a specific modulus (e.g., 64-bit signed or unsigned) rather than requiring use of the wrap functions and arithmetic separately.
-- Constants for the maximum and minimum 64-bit signed and unsigned integer
-
-### Any other numerical types, and generalization to binary data and value types
-
-In the course of development of ES2015, the proposal to add 64-bit integers was generalized significantly into a value types/binary data proposal. This big proposal became very complicated and unwieldy, so it was dropped from ES2015. The current proposal is much smaller, and could be built on incrementally. Value types and smaller integer types may be possible follow-ons, and this proposal is designed to generalize to those, but not block on them.
-
-## Implementation status
-
-- [V8](https://bugs.chromium.org/p/v8/issues/detail?id=6791) by Georg Neis and Jakob Kummerow
-- [JSC](https://bugs.webkit.org/show_bug.cgi?id=175359) by Caio Lima and Robin Morisset
-- [SpiderMonkey](https://bugzilla.mozilla.org/show_bug.cgi?id=1366287) by Robin Templeton
 
-## Specification
+### State of the Proposal
 
-See the [specification](https://tc39.github.io/proposal-bigint/) for more fine details.
+This proposal is currently in Stage 3. [Check out the TC39 stages to table to understand what that means.](https://tc39.github.io/process-document/)
```

Comment:
The process document isn't very friendly to read, but I guess it's all we have for now. Eventually, I think we should point this to a document explaining the issue as part of the how-we-work effort. (Optional editorial suggestion: you could just make "Stage 3" a link to the process document.)